app/controllers/admin/events_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+module Admin
+  class EventsController < ApplicationController
+    before_action :authenticate_user!
```

Comment:
@stephanieminn I added this, per your suggestion, and also added a test making sure that hitting this action without authenticating will return a 302 and redirect to login. Thanks for pointing this out!